doc/bridge_packages/flame_forge2d/joints.md

```diff
@@ -112,3 +112,45 @@ to 0, the higher the value, the less springy the joint becomes.
 ```{warning}
 Do not use a zero or short length.
 ```
+
+
+### `FrictionJoint`
+
+A `FrictionJoint` is used for top-down friction. It provides 2D translational friction and angular
+friction.
+
+The `initialize` method requires two bodies and an anchor. The first body is the body the friction
+will be applied to. The second body can be any body in the world, like a floor or an invisible
+body. The reason is that the friction force will be applied to the first body when it collides with
+any other bodies, not only the second body of the joint.
+
+The third parameter is the anchor point in the world where the friction force will be applied.
```

Comment:
could you comment how this evolves as the body moves?
for example you are providing the center of the ball in world coordinates as the third parameter in the example; when the ballBody moves, how does that affect the friction?

Reply:
It gets converted to a local coordinate inside `FrictionJointDef`. 

```
void initialize(A bodyA, B bodyB, Vector2 anchor) {
  this.bodyA = bodyA;
  this.bodyB = bodyB;
  localAnchorA.setFrom(bodyA.localPoint(anchor));
  localAnchorB.setFrom(bodyB.localPoint(anchor));
}
```

It's really hard to find any documentation in box2d on this parameter. From tests, it seems like the friction force is applied along the axis between two local anchors of the Joint.